build.sbt

```diff
@@ -17,7 +17,8 @@ libraryDependencies ++= Seq(
   "com.gu"                   %  "kinesis-logback-appender"    % "1.3.0",
   "com.gu"                   %% "pan-domain-auth-play_2-5"    % "0.4.1",
   "io.circe"                 %% "circe-parser"                % "0.7.0",
-  "net.logstash.logback"     %  "logstash-logback-encoder"    % "4.2"
+  "net.logstash.logback"     %  "logstash-logback-encoder"    % "4.2",
+  "com.squareup.okhttp"      % "okhttp"                       % "2.4.0"
```

Comment:
So the main reason is because it's been used in TagManager, Media Atom Maker etc the other projects, but I can switch it out